source/ios/AdaptiveCards/ADCIOSVisualizer/ADCIOSVisualizer.xcodeproj/project.pbxproj

```diff
@@ -105,6 +107,8 @@
 /* End PBXCopyFilesBuildPhase section */
 
 /* Begin PBXFileReference section */
+		F4071C801FD63BC700AF4FEA /* AdcBizCardSample.json */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = text.json; name = AdcBizCardSample.json; path = ../../../../../../Downloads/AdcBizCardSample.json; sourceTree = "<group>"; };
```

Comment:
This sample won't be found.

Reply:
thanks!
that is correct

---
In reply to: [155118369](https://github.com/Microsoft/AdaptiveCards/pull/1029#discussion_r155118369) [](ancestors = 155118369)